app/assets/stylesheets/pageflow/navigation_mobile.css.scss

```diff
@@ -99,6 +99,44 @@ body {
     }
   }
 
+  .parent_page {
+    @include background-icon-center($color: #ddd);
+    @include fa-arrow-left-icon;
+    cursor: pointer;
+    pointer-events: all;
+    position: absolute;
+    top: 16px;
+    right: 60px;
+    width: 45px;
+    height: 45px;
+    visibility: hidden;
+    @include transform(translate3d(0, -100px, 0));
+    @include transition(transform 0.4s ease, visibility 0.4s, right 250ms);
+
+    &.is_visible {
+      visibility: visible;
+      @include transform(translate3d(0, 0, 0));
+    }
+
+    &:before {
+      z-index: 2;
+    }
+
+    &:after {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 6px;
+      left: 6px;
+      width: 34px;
+      height: 34px;
+      border-radius: 17px;
+      background-color: rgba(53, 53, 53, 0.9);
```

Comment:
Color literals like `rgba(53, 53, 53, 0.9)` should only be used in variable declarations; they should be referred to via variable everywhere else.